lib/assets/fileList.tcl

```diff
@@ -0,0 +1,52 @@
+# DESCRIPTION:
```

Comment:
This is the TCL script, most if it is comments since this is new territory for me, but the instructions also show how to run the script outside of the Rails application which is really handy for testing and tweaking it.